omod/src/main/java/org/openmrs/module/coreapps/page/controller/findpatient/FindPatientPageController.java

```diff
@@ -28,6 +31,7 @@ public void get(PageModel model, @RequestParam("app") AppDescriptor app, UiSessi
         model.addAttribute("heading", app.getConfig().get("heading").getTextValue());
         model.addAttribute("label", app.getConfig().get("label").getTextValue());
         model.addAttribute("showLastViewedPatients", app.getConfig().get("showLastViewedPatients").getBooleanValue());
+		model.addAttribute("showRegistrationLink", app.getConfig().get("showRegistrationLink").getTextValue());
```

Comment:
But from what you are saying, it appears like two, you're saying that the value of the showRegistrationLink attribute is an appId, then there is the GP, see what am saying? Unless showRegistrationLink is a boolean to determine whether to display the registration button or not, do you get my confusion?